target/linux/ramips/mt76x8/base-files/etc/board.d/01_leds

```diff
@@ -52,11 +52,7 @@ tama,w06)
 	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth0"
 	ucidef_set_led_wlan "wlan" "WLAN" "$boardname:green:wlan" "phy0tpt"
 	;;
-tplink,archer-c20-v4)
-	ucidef_set_led_switch "lan" "lan" "$boardname:green:lan" "switch0" "0x1e"
-	ucidef_set_led_switch "wan" "wan" "$boardname:green:wan" "switch0" "0x01"
-	ucidef_set_led_netdev "wlan2g" "wlan2g" "$boardname:green:wlan2g" "wlan0"
-	;;
+tplink,archer-c20-v4|\
```

Comment:
If there are tpt riggers available, I'd assume both LEDs can be triggered in DTS?

Reply:
done. thanks!